src/internal/dedup_address_type.hpp

```diff
@@ -13,24 +13,32 @@
 #include <blocksci/core/meta.hpp>
 
 #define DEDUP_ADDRESS_TYPE_LIST VAL(SCRIPTHASH), VAL(PUBKEY), VAL(MULTISIG), VAL(NULL_DATA), VAL(WITNESS_UNKNOWN), VAL(NONSTANDARD)
+// DEDUP_ADDRESS_TYPE_SET is never used
 #define DEDUP_ADDRESS_TYPE_SET VAL(SCRIPTHASH) VAL(PUBKEY) VAL(MULTISIG) VAL(NULL_DATA) VAL(WITNESS_UNKNOWN) VAL(NONSTANDARD)
+
 namespace blocksci {
-    
-    
+
     struct DedupAddressType {
         
         enum Enum {
+            // after preprocessing: NONSTANDARD, PUBKEY, SCRIPTHASH, MULTISIG, NULL_DATA, WITNESS_UNKNOWN
```

Comment:
The order here has recently changed (in #0f4854), nonstandard is now the last

Reply:
Fixed in 77b2115.